src/pages/home/home.tsx

```diff
@@ -649,26 +321,14 @@ const HomePage = () => {
       const sessionData = await walletService.retrieveCurrentSession();
       const currentAsset = await walletService.retrieveDefaultWalletAsset(sessionData);
       const allAssets = await walletService.retrieveCurrentWalletAssets(sessionData);
-
-      const allNftTransfer: NftAccountTransactionData[] = await walletService.getAllNFTAccountTxs(
-        sessionData,
-      );
-
       const allNFTs: NftModel[] = await walletService.retrieveNFTs(sessionData.wallet.identifier);
+
       const currentNftList = processNftList(allNFTs);
       setProcessedNftList(currentNftList);
       setNFTList(allNFTs);
-
       setdefaultWalletAsset(currentAsset);
-
-      const nftTransferTabularData = convertNftTransfers(allNftTransfer);
-
-      showWalletStateNotification(sessionData.wallet.config);
-      checkNewlyAddedStaticAssets(sessionData);
-
-      // setTransfers(transferTabularData);
-      setNftTransfers(nftTransferTabularData);
-      // setUserAsset(currentAsset);
+      showWalletStateNotification(currentSession.wallet.config);
+      checkNewlyAddedStaticAssets(currentSession);
```

Comment:
```suggestion
      showWalletStateNotification(sessionData.wallet.config);
      checkNewlyAddedStaticAssets(sessionData);
```

Inside `useEffect` here, `currentSession` is not set yet, it's better to use `sessionData` which is guaranteed to have been set.